java/component/keyple-core/src/main/java/org/eclipse/keyple/core/plugin/AbstractPlugin.java

```diff
@@ -56,9 +60,11 @@ public final String getName() {
    * threaded plugin {@link AbstractThreadedObservablePlugin}
```

Comment:
This comment is no longer valid.
The map is now initialized when registering the plugin.